jujugui/static/gui/src/app/views/utils.js

```diff
@@ -621,8 +621,18 @@ YUI.add('juju-view-utils', function(Y) {
     @param {Object} db Reference to the app db.
   */
   utils.exportEnvironmentFile = function(db, legacyServicesKey) {
-    var result = db.exportDeployer(legacyServicesKey);
+    const apps = db.services.toArray();
+    const idMap = new Map();
+    // Store a map of all the temporary app ids to the real ids.
+    apps.forEach(app => {
+      idMap.set(app.get('id'), app.get('name'));
+    });
+    var result = db.exportDeployer(legacyServicesKey, true);
```

Comment:
You have added the option for exporting uncommitted but then hard coded the value to `true` Do we maybe want this to be user definable when they click export and there are uncommitted items?